code/game/machinery/Sleeper.dm

```diff
@@ -163,41 +163,107 @@
 		dat += text("<BR><BR><A href='?src=\ref[];mach_close=sleeper'>Close</A>", user)
 		user << browse(dat, "window=sleeper;size=400x500")
 		onclose(user, "sleeper")
+		*/

```

Comment:
Was leaving this there so that I could see what old features might need to be spliced back in, but I'll cut that out the next commit.